Library/Formula/corectl.rb

```diff
@@ -0,0 +1,42 @@
+require "language/go"
+
+class Corectl < Formula
+  desc "CoreOS over OS X made very simple"
+  homepage "https://github.com/TheNewNormal/corectl"
+  url "https://github.com/TheNewNormal/corectl/archive/v0.0.19.tar.gz"
+  sha256 "323ebe7e3f612a2656e4b844793f62433a46604ee78455446e04db6b8bc0247e"
+  head "https://github.com/TheNewNormal/corectl.git", :branch => "golang"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+  depends_on "gnu-sed" => :build
+  depends_on :macos => :yosemite
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    mkdir_p buildpath/"src/github.com/TheNewNormal/"
+    ln_s buildpath, buildpath/"src/github.com/TheNewNormal/#{name}"
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    # args = []
+    # args << "VERSION=#{version}" if build.stable?
+    # system "make", "corectl", *args
+    # busts with "cannot load DWARF output from ""...
+    # similar to https://github.com/jwaldrip/homebrew-utils/issues/1
+    # workaround ...
+    cmd = `make homebrewHack VERSION=#{version}` if build.stable?
+    cmd = `make homebrewHack` if build.head?
```

Comment:
Ok, I see a bit more why this may be needed. That said, I'd rather fix this properly in Homebrew so this hack isn't needed. It seems to be related to `cgo` trying to build debug symbols which Homebrew strips. Do you know if there's any way of convincing `cgo` to not build DWARF symbols? Thanks!